frontend/src/state/actions/schedulesActions.ts

```diff
@@ -1,21 +1,4 @@
 import { createAction } from "typesafe-actions";
-import { PastPresentSchedule } from "../../models/types";
-
-export const addScheduleAction = createAction(
-  "schedules/ADD_SCHEDULE",
-  (schedule: PastPresentSchedule, name: string) => ({
-    schedule,
-    name,
-  })
-)();
-
-export const removeScheduleAction = createAction(
-  "schedules/REMOVE_SCHEDULE",
-  (schedule: PastPresentSchedule, name: string) => ({
-    schedule,
-    name,
-  })
-)();
```

Comment:
You removed these from here but I think you still refer to addScheduleAction and removeScheduleAction in the reducer and the SwitchPlanPopper so it's not deploying properly